### DISCUSSION TITLE
Web Framework Benchmarks

### DISCUSSION THREAD

**Discussion by Malix-Labs** [denoland/deno#25003] [Q&A] [answered]
Hey !

As [Bun's WebSockets uses uWebSockets](https://bun.sh/docs/api/websockets#:~:text=Internally%20Bun%27s%20WebSocket%20implementation%20is%20built%20on%20uWebSockets.)

And given that [uWebSockets is the second best WebSocket library for JavaScript](https://www.techempower.com/benchmarks/#hw=ph&test=fortune&section=data-r22&l=zik0sf-cn3)

Does Deno also plan to implement it ?

**Comment by bartlomieju** [comment, marked as answer]
No, we don't plan to use uWebSockets. We have our own library called https://github.com/denoland/fastwebsockets.

**Reply by Malix-Labs** [reply]
Are there any comparison benchmarks ?

**Reply by bartlomieju** [reply]
@littledivy do you have any benchmarks?

**Reply by littledivy** [reply]
https://github.com/denoland/fastwebsockets/tree/main/benches

**Reply by Malix-Labs** [reply]
Thanks !

I also published https://github.com/TechEmpower/FrameworkBenchmarks/issues/9207
